docs/content/guides/accessibility/accessibility.md

```diff
@@ -0,0 +1,185 @@
+---
+id: o4qhm1bg
+title: Handsontable - Accessibility
+metaTitle: Accessibility - JavaScript Data Grid | Handsontable
+description: Learn about Handsontable's accessibility features.
+permalink: /accessibility
+canonicalUrl: /accessibility
+tags:
+  - accessibility
+  - a11y
+  - aria
+  - jaws
+  - nvda
+  - voiceover
+  - wcag
+  - section 508
+  - ada
+  - compliance
+  - vpat
+  - acr
+react:
+  id: x82phf34
+  metaTitle: Accessibility - React Data Grid | Handsontable
+searchCategory: Guides
+---
+[[toc]]
+
+## Overview
+
+Handsontable offers comprehensive accessibility support, including built-in keyboard navigation in line with international standards. Our design ensures that individuals with disabilities can fully engage with Handsontable.
+
+Features of Handsontable include:
```

Comment:
I think the title "Features of Handsontable" could be confusing, as this is purely a subset of all of Handsontables features, I think we need a title that outlines that these are our accessibility features.

I will also in the future go through the content of each dot point and add more or give some suggestions

Reply:
changed